cicd/build/monit-python/monit_build2.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
@novicecpp do we prefer this file or `cicd/build/monit-python/monit_build.sh` ? :) 
(sorry, I forgot that i could copy how you write dockerfiles for pypi :) )

Reply:
Sorry, I am not understand the question. Maybe it is not relevant to the latest change anymore?

Reply:
I wrote `monit_build.sh` before i noticed that I did not need to copy files in a subdirectory. I could just build the docker container from the dmwm/CRABServer root and use that directory as build as context. I will clean this up, assuming that you prefer `monit_build2.sh` more than `monit_build.sh`, as I do now, some weeks after creating it  :)

Reply:
so that directory will eventually have only one `Dockerfile` and only one `monit_build.sh` file ?